assets/xml/objects/object_oF1d_map.xml

```diff
@@ -1,108 +1,142 @@
+        <Texture Name="gGoronRolledUpHalfSkinTex" OutName="goron_rolled_up_half_skin" Format="ci8" Width="16" Height="16" Offset="0x80F0" />
+        <Texture Name="gGoronRolledUpHalfBellyButtonTex" OutName="goron_rolled_up_half_belly_button" Format="ci8" Width="8" Height="8" Offset="0x81F0" />
+        <Texture Name="gGoronRolledUpHalfBottomTex" OutName="goron_rolled_up_half_bottom" Format="ci8" Width="8" Height="16" Offset="0x8230" />
+        <Texture Name="gGoronRolledUpHalfRocksTex" OutName="goron_rolled_up_half_rocks" Format="ci8" Width="32" Height="64" Offset="0x82B0" />
+        <Texture Name="gGoronRolledUpHalfHairTex" OutName="goron_rolled_up_half_hair" Format="ci8" Width="16" Height="16" Offset="0x8AB0" />
+
+        <!-- Rolled Up -->
+        <DList Name="gGoronEmpty2DL" Offset="0x91A0" />
+        <DList Name="gGoronRolledUpDL" Offset="0x91A8" />
+        <Texture Name="gGoronRolledUpTLUT" OutName="goron_rolled_up_tlut" Format="rgba16" Width="16" Height="16" Offset="0x94E0" />
+        <Texture Name="gGoronRolledUpSkinTex" OutName="goron_rolled_up_skin" Format="ci8" Width="16" Height="16" Offset="0x96E0" />
+        <Texture Name="gGoronRolledUpBellyButtonTex" OutName="goron_rolled_up_belly_button" Format="ci8" Width="8" Height="8" Offset="0x97E0" />
+        <Texture Name="gGoronRolledUpBottomTex" OutName="goron_rolled_up_bottom" Format="ci8" Width="8" Height="16" Offset="0x9820" />
+        <Texture Name="gGoronRolledUpRocksTex" OutName="goron_rolled_up_stones" Format="ci8" Width="32" Height="64" Offset="0x98A0" />
+
+        <!-- Animation -->
+        <Animation Name="gGoronTPoseAnim" Offset="0xA118" />
+
+        <!-- DLists -->
+        <DList Name="gGoronBodyDL" Offset="0xCC50" />
+        <DList Name="gGoronHeadDL" Offset="0xCF70" />
+        <DList Name="gGoronRightUpperArmDL" Offset="0xD688" />
+        <DList Name="gGoronRightForearmDL" Offset="0xD808" />
+        <DList Name="gGoronRightHandDL" Offset="0xD938" />
+        <DList Name="gGoronLeftUpperArmDL" Offset="0xDBB8" />
+        <DList Name="gGoronLeftForearmDL" Offset="0xDD38" />
+        <DList Name="gGoronLeftHandDL" Offset="0xDE68" />
+        <DList Name="gGoronPelvisDL" Offset="0xE070" />
+        <DList Name="gGoronRightThighDL" Offset="0xE130" />
+        <DList Name="gGoronRightShinDL" Offset="0xE218" />
+        <DList Name="gGoronRightFootDL" Offset="0xE348" />
+        <DList Name="gGoronLeftThighDL" Offset="0xE610" />
+        <DList Name="gGoronLeftShinDL" Offset="0xE6F8" />
+        <DList Name="gGoronLeftFootDL" Offset="0xE828" />
+
+        <!-- Textures -->
+        <Texture Name="gGoronTLUT" OutName="goron_tlut" Format="rgba16" Width="16" Height="16" Offset="0xEA78" />
+        <Texture Name="gGoronEyeTLUT" OutName="goron_eye_tlut" Format="rgba16" Width="16" Height="16" Offset="0xEC78" />
+        <Texture Name="gGoronSkinTex" OutName="goron_skin" Format="ci8" Width="16" Height="16" Offset="0xEE78" />
```

Comment:
I'm not that convinced that Gorons have skin, but I don't have a better suggestion.